File: mock/1.cpp
#include "../common.h"


class Solution {
public:
    vector<string> findOcurrences(string text, string first, string second) {
        string s = first + " " + second;
        vector<string> v;
        
        int p = text.find(s);
        while(p != string::npos){
            
        }
    }
};

int main(){
    string str("alice is a good girl she is a good student");
    string w1("a");
    string w2("good");

    Solution s;
    auto r = s.findOcurrences(str, w1, w2);
    for(auto &w:r)
        cout << w << endl;



}

File: mock/1.py
class Solution:
    def findOcurrences(self, text, first, second):
        l = text.split(" ")
        rst = []
        if len(l) < 3:
            return rst
        for i in range(len(l)-2):
            if l[i] == first and l[i+1] == second:
                rst.append(l[i+2])
        return rst


text =  "alice is a good girl she is a good student"
first = "a"
second = "good"

s = Solution()
print(s.findOcurrences(text, first, second))
